### DISCUSSION TITLE
关于packages包里的network已经很老了，ipv6相关的也一直没更新，老是出问题，请问是为啥不随openwrt项目一起更新呢？

### DISCUSSION THREAD

**Discussion by sss132919** [coolsnowwolf/lede#11898] [Bugs / 问题反馈]
关于packages包里的network已经很老了，ipv6相关的也一直没更新，老是出问题，请问是为啥不随openwrt项目一起更新呢？

**Comment by liwenjie119** [comment]
的确希望能更新，因为ipv6相关我的确遇到问题了

**Comment by Travel2Here** [comment]
+1

**Comment by kxdn3** [comment]
ipv6确实是…

**Comment by GreatMichaelLee** [comment]
@coolsnowwolf 大佬能不能更新一下，slaac ra的设置luci没有界面，dhcpv6的lan设备静态指定hostid配下去也不是想要的...

**Comment by coolsnowwolf** [comment]
我这里没有测试 ipv6 的环境，你们可以测试后发个 pr 更新即可

**Reply by GreatMichaelLee** [reply]
> 我这里没有测试 ipv6 的环境，你们可以测试后发个 pr 更新即可

我有环境测，就是不会更啊....
